### ISSUE TITLE
Connect to Schema Registry over SSL

### ISSUE THREAD

**Issue by evaisman-tc** [tchiotludo/akhq#685]
Hey folks, I was wondering if anyone has had luck with connecting with schema registry over ssl. 

The configurations I'm using are as such:
```
  connections:
    kafka:
      properties:
        bootstrap.servers: "kafkadev:9093"
        security.protocol: SSL
        ssl.truststore.location: /alloc/certs/truststore.jks
        ssl.truststore.password: <pass>
        ssl.keystore.location: /alloc/certs/keystore.jks
        ssl.keystore.password: <pass>
        ssl.key.password: <pass>
        ssl.endpoint.identification.algorithm: ""
      schema-registry:
        url: "https://schema-registry:8082"
        properties: # standard kafka properties (optional)
          schema.registry.ssl.truststore.location: /alloc/certs/truststore.jks
          schema.registry.ssl.truststore.password: <pass>
          schema.registry.ssl.keystore.location: /alloc/certs/keystore.jks
          schema.registry.ssl.keystore.password: <pass>
          schema.registry.ssl.key.password: <pass>
      connect:
        - name: connect-1
          url: "https://kconnect:8083"
          basic-auth-username: username
          basic-auth-password: <pass>
          ssl-trust-store: /alloc/certs/truststore.jks
          ssl-trust-store-password: <pass>
          ssl-key-store: /alloc/certs/keystore.jks
          ssl-key-store-password: <pass>
```

The rest of akhq is able to connect as expected to both kafka and kconnect.

I tried following the directions in the README as well as the issue here: https://github.com/tchiotludo/akhq/issues/88#issuecomment-575665149 and the confluent docs here https://docs.confluent.io/platform/current/schema-registry/security/index.html#additional-configurations-for-https but no luck :/ Any ideas?

akhq version: tchiotludo/akhq:0.17.0

stacktrace when visiting `ui/kafka/schema`:

```
Internal Server Error: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

javax.net.ssl.SSLHandshakeException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at java.base/sun.security.ssl.Alert.createSSLException(Unknown Source)
	at java.base/sun.security.ssl.TransportContext.fatal(Unknown Source)
	at java.base/sun.security.ssl.TransportContext.fatal(Unknown Source)
	at java.base/sun.security.ssl.TransportContext.fatal(Unknown Source)
	at java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.checkServerCerts(Unknown Source)
	at java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.onConsumeCertificate(Unknown Source)
	at java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.consume(Unknown Source)
	at java.base/sun.security.ssl.SSLHandshake.consume(Unknown Source)
	at java.base/sun.security.ssl.HandshakeContext.dispatch(Unknown Source)
	at java.base/sun.security.ssl.HandshakeContext.dispatch(Unknown Source)
	at java.base/sun.security.ssl.TransportContext.dispatch(Unknown Source)
	at java.base/sun.security.ssl.SSLTransport.decode(Unknown Source)
	at java.base/sun.security.ssl.SSLSocketImpl.decode(Unknown Source)
	at java.base/sun.security.ssl.SSLSocketImpl.readHandshakeRecord(Unknown Source)
	at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source)
	at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source)
	at java.base/sun.net.www.protocol.https.HttpsClient.afterConnect(Unknown Source)
	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at java.base/java.net.HttpURLConnection.getResponseCode(Unknown Source)
	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(Unknown Source)
	at io.confluent.kafka.schemaregistry.client.rest.RestService.sendHttpRequest(RestService.java:272)
	at io.confluent.kafka.schemaregistry.client.rest.RestService.httpRequest(RestService.java:351)
	at io.confluent.kafka.schemaregistry.client.rest.RestService.getAllSubjects(RestService.java:805)
	at io.confluent.kafka.schemaregistry.client.rest.RestService.getAllSubjects(RestService.java:788)
	at org.akhq.repositories.SchemaRegistryRepository.all(SchemaRegistryRepository.java:68)
	at org.akhq.repositories.SchemaRegistryRepository.list(SchemaRegistryRepository.java:42)
	at org.akhq.controllers.SchemaController.list(SchemaController.java:51)
	at org.akhq.controllers.$SchemaControllerDefinition$$exec1.invokeInternal(Unknown Source)
	at io.micronaut.context.AbstractExecutableMethod.invoke(AbstractExecutableMethod.java:146)
	at io.micronaut.context.DefaultBeanContext$4.invoke(DefaultBeanContext.java:474)
	at io.micronaut.web.router.AbstractRouteMatch.execute(AbstractRouteMatch.java:312)
	at io.micronaut.web.router.RouteMatch.execute(RouteMatch.java:118)
	at io.micronaut.http.server.netty.RoutingInBoundHandler.lambda$buildResultEmitter$10(RoutingInBoundHandler.java:1363)
	at io.reactivex.internal.operators.flowable.FlowableDefer.subscribeActual(FlowableDefer.java:35)
	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
	at io.reactivex.internal.operators.flowable.FlowableSubscribeOn$SubscribeOnSubscriber.run(FlowableSubscribeOn.java:82)
	at io.reactivex.internal.schedulers.ExecutorScheduler$ExecutorWorker$BooleanRunnable.run(ExecutorScheduler.java:288)
	at io.reactivex.internal.schedulers.ExecutorScheduler$ExecutorWorker.run(ExecutorScheduler.java:253)
	at io.micrometer.core.instrument.composite.CompositeTimer.record(CompositeTimer.java:79)
	at io.micrometer.core.instrument.Timer.lambda$wrap$0(Timer.java:148)
	at io.micronaut.scheduling.instrument.InvocationInstrumenterWrappedRunnable.run(InvocationInstrumenterWrappedRunnable.java:47)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at java.base/sun.security.validator.PKIXValidator.doBuild(Unknown Source)
	at java.base/sun.security.validator.PKIXValidator.engineValidate(Unknown Source)
	at java.base/sun.security.validator.Validator.validate(Unknown Source)
	at java.base/sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source)
	at java.base/sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source)
	at java.base/sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source)
	... 43 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at java.base/sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source)
	at java.base/sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source)
	at java.base/java.security.cert.CertPathBuilder.build(Unknown Source)
	... 49 more
```
Thanks!

**Comment by alselies** [comment]
after trying any config barely testing by comment any line it seems akhq not providing any truststore even using schema-registry.properties config, i don't why, may be some one can answer this. i'm still debuging this.

if u want to see ssl debug try this command : nohup java -Djavax.net.debug=ssl,handshake -Dmicronaut.config.files=/data1/platform/akhq/application.yml -jar akhq.jar &> akhq.out &

check out akhq.out and u will see if any server certificate is match with your truststore or not. in my case the certificate match but it seems akhq not providing any truststore. TwT

**Comment by alozano3** [comment]
@evaisman-tc and @alselies , the 0.17.0 version does not work with Schema Registry and SSL. That problem was solved in #603. Trey qith the dev version tchiotludo/akhq:dev

**Comment by evaisman-tc** [comment]
Much appreciated, thank you!
